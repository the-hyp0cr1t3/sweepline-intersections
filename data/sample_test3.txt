3
1 1 9 9
2 3 7 8
3 2 7 8