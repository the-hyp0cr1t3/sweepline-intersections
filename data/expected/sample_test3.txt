2
5.0000000000 5.0000000000 1 3
7.0000000000 8.0000000000 2 3

